apps/src/music/analytics/AnalyticsReporter.ts

```diff
@@ -165,6 +168,19 @@ export default class AnalyticsReporter {
     track(eventType, payload).promise;
   }
 
+  onSoundsPlayed(id: string) {
+    if (!this.sessionInProgress) {
+      this.log('No session in progress');
+      return;
+    }
+
+    if (!this.soundsPlayed[id]) {
+      this.soundsPlayed[id] = 1;
+    } else {
+      this.soundsPlayed[id]++;
+    }
```

Comment:
not blocking, and I'm not convinced this is any better, but as an alternative:

```
this.soundsPlayed[id] = 1 + (this.soundsPlayed[id] ?? 0)
```

clearly I've just got nullish coalescing operators on my mind this morning. 😄

Reply:
Sure - I can update! I was first introduced to nullish coalescing operators by Cassi. https://github.com/code-dot-org/code-dot-org/pull/58086/files#r1574847284
Thanks!